common-model/src/main/java/com/sequenceiq/common/model/CloudStorageCdpService.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.common.model;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public enum CloudStorageCdpService {
+
+    ZEPPELIN_SERVER,
```

Comment:
I think are comming for a josn configuration. do we need this enum ?

Reply:
- contra against this: the enum has to store the same values as the json, so we have to maintain it at 2 places
- pro: it is straightforward in the code that what kind of values can be accepted in the API and what we store in the database

I think the pro argument is stronger, what do you think?

Reply:
Actually this enum is even used for parsing the JSON. This makes the contra argument weaker.